app/controllers/comments_controller.rb

```diff
@@ -75,7 +75,13 @@ def create
         render json: { status: "comment already exists" }
         return
       end
-
+      if current_user.experience_level
+        RatingVote.create(article_id: @comment.commentable_id,
```

Comment:
I don't agree that commenting on an article should creating an experience level rating based on the user's experience level. Our own mod messages encourage this behavior across experience levels:

> @new_user just left a comment. Since they are new to the community, could you leave a nice reply to help them feel welcome? Thank you!

Reply:
Yeah that's a good call